balena/models/application.py

```diff
@@ -517,10 +554,108 @@ def set_to_release(self, app_id, commit_id):
         }
 
         data = {
-            'commit': commit_id
+            'commit': full_release_hash,
+            'should_track_latest_release': False
         }
 
         return self.base_request.request(
             'application', 'PATCH', params=params, data=data,
             endpoint=self.settings.get('pine_endpoint')
         )
+
+    def will_track_new_releases(self, app_id):
+        """
+        Get whether the application is configured to receive updates whenever a new release is available.
+
+        Args:
+            app_id (str): application id.
+
+        Returns:
+            bool: is tracking the latest release.
+
+        Examples:
+            >> > balena.models.application.will_track_new_releases('5685')
+            True
+
+        """
+
+        return bool(self.get_by_id(app_id)['should_track_latest_release'])
+
+    def is_track_latest_release(self, app_id):
```

Comment:
I think better names for this one and the above are:

- is_track_latest_release  -> is_on_latest_release
- will_track_new_releases  -> is_tracking_latest_release

Reply:
@dfunckt i have the same thought but I think it's better to respect and use the same name for same methods with the JS SDK.

Reply:
@nghiant2710 where are these defined on JS SDK? Github search doesn't bring up any result.

Reply:
Oh, NVM, I search for the exact strings, but on JS they're camelCase.

Reply:
https://github.com/balena-io/balena-sdk/blob/master/lib/models/application.coffee#L683

Reply:
@dfunckt ah yes, my typo, the same method in device class is correct but this one is wrong. Fixed 👍